src/Management/src/Endpoint/CloudFoundry/CFContext.cs

```diff
@@ -6,9 +6,9 @@
 
 namespace Steeltoe.Management.Endpoint.CloudFoundry;
 
-internal class CFContext : IContextName
+internal sealed class CFContext : IContextName
```

Comment:
Yes we can have multiple at the same time - so its not dependent simply on IsCloudFoundry. It depends on which extension methods were used to addActuators . By default AddAllActuators when ISCloudfoundry will add both contexts. I can see something similar being required in TAP - althought at the moment its not required. We could probably change that to an enum, because any code we write will have to handle that. More interesting is someone trying to extend this functionality as a consumer of steeltoe.

Reply:
I'm not too concerned with external developers trying to extend Steeltoe for another cloud platform, because it requires special-cased code at so many places. I would go with a simple internal `[Flags]` enum.

Suggestion:
```c#
[Flags]
internal enum ActuatorPlatforms
{
  None = 0,
  Default = 1,
  CloudFoundry = 1 << 1,
  All = Default | CloudFoundry
}
```

@TimHess thoughts?

Reply:
foreach (ActuatorPlatforms platform in Enum.GetValues<ActuatorPlatforms>())
        if (_managementOptions.CurrentValue.ActuatorPlatforms.HasFlag(platform))
   
vs   
    foreach (EndpointContext context in Enum.GetValues<EndpointContexts>())
        if (_managementOptions.CurrentValue.EndpointContexts.HasFlag(context))

        I think this enum is the name of NamedIOptions and represents supported root paths. While it has something to do with platforms in that CF has an extra context path /cloudfoundryapplication, that is not the most important thing about this name. Other options for names are EndpointRoot, ContextPaths

Reply:
It seemed like Flags was getting in our way, so I removed the flags annotation - so this recommendation may not apply anymore?